src/components/modules/blockSelection.ts

```diff
@@ -286,24 +286,24 @@ export default class BlockSelection extends Module {
    *
    * @returns {Promise<void>}
    */
-  public copySelectedBlocks(e: ClipboardEvent): Promise<void> {
+  public async copySelectedBlocks(e: ClipboardEvent): Promise<void> {
```

Comment:
That would break copying in FireFox as we can't change clipboard data in async functions, so `saveToHTML` need to be synchronous